### DISCUSSION TITLE
3D Assembly guide -

### DISCUSSION THREAD

**Discussion by phovorka** [google/model-viewer#4595] [Show and tell]
Hi,
i would like to share with you my concept of 3D ASSEMBLY instruction based on model-viewer technology. 
This concept was shown to IKEA manager and he was excited from it. 
[https://assembly3d.glitch.me/](https://assembly3d.glitch.me/)

I have some questions and i would like to hear some feedback from audience here........
- Do you think this could be a good idea to make a start-up?
- Would someone like to join me ? Make it real product.... (backend- front end people, sales for US teritory)  
- where to find investor for such a idea?

all ideas and tips are welcome

Dont look on the source code :-) it was written fast and its ugly ... its only proof of concept.

**Vision:**
NEW ASSEMBLY INSTRUCTION in 3D instead of ugly papers and PDFs. Accessible on all devices.

Cheers Petr

**Comment by vehrmann** [comment]
Really great work!
I actually sell a [wall desk](https://vidajo.de/produkt/wandschreibtisch-wall-ti-100-cm/) and would consider such a tool as an addition to our manual.
At the moment I am writing my own product configurator with the help of model-viewer. To be released in a couple of days, I will post here in the show-and-tell section.

To answer your questions:

- Do you think this could be a good idea to make a start-up?
Yes, the idea is great, but before digging for gold, you should know the market and if someone is already doing a similar thing.
I don't know how much you are into entrepreneurship, but you need to do proper research and business planning before actually starting. Solely the (indeed good) idea is not enough.
- Would someone like to join me ? Make it real product.... (backend- front end people, sales for US teritory)
Be open about the market! As it would be a digital product, you could offer it globally.
In case you want to chat about it (and also some topics and issues I encountered with product development, manuals etc.), just contact me. I am no professional programmer, but can produce working code to some extend :)
- where to find investor for such a idea?
This is a rather general questions, dependent on the type of investor you want. Only finance? Someone with a helpful network? Someone actually helping you developing the product?

Best
Vincent

**Reply by phovorka** [reply]
Hi Vincent, thank you for your feedback. I am looking forward for your product configurator.

**Comment by bysmoke** [comment]
How is your project?

**Comment by phovorka** [comment]
[www.assembly3d.cz ](https://www.assembly3d.cz/) is alive!
